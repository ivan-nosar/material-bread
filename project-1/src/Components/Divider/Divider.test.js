import React from 'react';
import { Divider } from '../../';

import renderer from 'react-test-renderer';

test('Divider Renders', () => {
  const tree = renderer.create(<Divider />).toJSON();
  expect(tree).toMatchSnapshot();
});
