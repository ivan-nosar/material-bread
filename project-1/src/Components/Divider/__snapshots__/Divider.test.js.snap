// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Divider Renders 1`] = `
<View
  style={
    Array [
      Object {
        "backgroundColor": "rgba(0,0,0,.12)",
        "width": "100%",
      },
      Object {
        "height": 0.5,
        "marginVertical": 8,
      },
      undefined,
    ]
  }
/>
`;
